src/opm/output/eclipse/AggregateNetworkData.cpp

```diff
@@ -0,0 +1,752 @@
+    iBran[Ix::UpTreeNode] = (uptr_nd_res.first) ? uptr_nd_res.second + 1 : 0 ;
+
+    iBran[Ix::VfpTableNo] = (branch.vfp_table().has_value()) ? branch.vfp_table().value() : 0;
+}
+} // Ibran
+
+namespace INobr {
+
+
+Opm::RestartIO::Helpers::WindowedArray<int>
+allocate(const std::vector<int>& inteHead)
+{
+    using WV = Opm::RestartIO::Helpers::WindowedArray<int>;
+    int nitPrWin = std::max(static_cast<int>(entriesPerInobr(inteHead)), 1);
+    return WV {
+        WV::NumWindows{ 1 },
+        WV::WindowSize{ static_cast<std::size_t>(nitPrWin)  }
+    };
+}
+
+template <class INobrArray>
+void staticContrib(const std::vector<int>&   inbr,
+                   INobrArray&  iNobr)
+{
+    for (std::size_t inb = 0; inb < inbr.size(); inb++) {
+        iNobr[inb] = inbr[inb];
+    }
+
+}
+} // Inobr
+
+namespace ZNode {
+std::size_t entriesPerZnode(const std::vector<int>& inteHead)
+{
+    return inteHead[Opm::RestartIO::Helpers::VectorItems::NZNODE];
+}
+
+Opm::RestartIO::Helpers::WindowedArray<
+Opm::EclIO::PaddedOutputString<8>
+>
+allocate(const std::vector<int>& inteHead)
+{
+    using WV = Opm::RestartIO::Helpers::WindowedArray<
+               Opm::EclIO::PaddedOutputString<8>
+               >;
+
+    return WV {
+        WV::NumWindows{ nodmax(inteHead) },
+        WV::WindowSize{ entriesPerZnode(inteHead) }
+    };
+}
+
+template <class ZNodeArray>
+void staticContrib(const std::string&       nodeName,
+                   ZNodeArray&               zNode)
+{
+    zNode[0] = nodeName;
+}
+} // Znode
+
+namespace RNode {
+std::size_t entriesPerRnode(const std::vector<int>& inteHead)
+{
+    return inteHead[Opm::RestartIO::Helpers::VectorItems::NRNODE];
+}
+
+Opm::RestartIO::Helpers::WindowedArray<double>
+allocate(const std::vector<int>& inteHead)
+{
+    using WV = Opm::RestartIO::Helpers::WindowedArray<double>;
+
+    return WV {
+        WV::NumWindows{ nodmax(inteHead) },
+        WV::WindowSize{ entriesPerRnode(inteHead) }
+    };
+}
+
+template <class RNodeArray>
+void dynamicContrib(const Opm::Schedule&      sched,
+                    const Opm::SummaryState&  sumState,
+                    const std::string&        nodeName,
+                    const std::size_t         lookup_step,
+                    const Opm::UnitSystem&    units,
+                    RNodeArray&               rNode)
+{
+    using Ix = ::Opm::RestartIO::Helpers::VectorItems::RNode::index;
+    // node dynamic pressure
+    std::string compKey = "GPR:" + nodeName;
+    rNode[Ix::NodePres] = (sumState.has(compKey)) ? sumState.get(compKey) : 0.;
+
+    // equal to 0. for fixed pressure nodes, 1. otherwise
+    rNode[Ix::FixedPresNode] = (fixedPressureNode(sched, nodeName, lookup_step)) ? 0. : 1.;
+
+    // equal to i) highest well p_thp if wellgroup and ii) pressure of uptree node with fixed pressure
+    rNode[Ix::PressureLimit] = nodePressure(sched, sumState, nodeName, units, lookup_step);
+
+    // fixed value
+    rNode[15] = 1.;
```

Comment:
15?

Reply:
The meaning of the value of this index is unknown, but it has to be 1.!